azure-pipelines.yml

```diff
@@ -26,3 +26,28 @@ jobs:
   parameters:
     name: Windows
     vmImage: vs2017-win2016
+
+- job: py37_32bit
+  pool:
+    vmImage: ubuntu-18.04
+
+  steps:
+    - script: |
+        docker pull quay.io/pypa/manylinux2014_i686
+        docker run -v $(pwd):/pandas quay.io/pypa/manylinux2010_i686 \
```

Comment:
This is 2010 here but 2014 on the line above - do they not need to match?

Reply:
It's a typo. I'll fix it.